### PR TITLE
Use yarn in CI

### PR DESCRIPTION
**Pull request by watrool**
Connected to https://github.com/rangle/rangle-starter/issues/189

### CODE REVIEW

**Comment by winkerVSbecks** [comment]
@watrool any update on this?

**Comment by watrool** [comment]
@winkerVSbecks no updates from my side.

File: circle.yml [hunk 1]
```diff
@@ -11,25 +11,31 @@ dependencies:
...
-    #- curl -O https://www.browserstack.com/browserstack-local/BrowserStackLocal-linux-x64.zip
-    #- unzip ./BrowserStackLocal-linux-x64.zip -d browserstack
-    #- rm ./BrowserStackLocal-linux-x64.zip
-    #- touch ./e2e/robot/results/browserstack.txt
-    #- ./browserstack/BrowserStackLocal -force -forcelocal REPLACE_THIS_WITH_BROWSERSTACK_KEY &> ./e2e/robot/results/browserstack.txt:
-    #    background: true
-    #- sleep 5
-    #- ./e2d/robot/exec/run_all_on_browserstack_desktop_ie.bash
-    #- ./e2e/robot/exec/run_smoke_on_local_chrome.bash
+    - curl -O https://www.browserstack.com/browserstack-local/BrowserStackLocal-linux-x64.zip
```
Comment (SethDavenport) [inline circle.yml]: Hmmm.... no we should figure out why it's timing out.

Comment (SethDavenport) [inline circle.yml]: I would prefer to run the tests against localhost (the ci container) than browserstack by default.

**Comment by watrool** [comment]
@SethDavenport Just update this PR with the official way to enable yarn on circle CI, and I also restored the e2e test part (to be same as the master)

**Comment by SethDavenport** [comment]
:shipit:

**Pull request merged**